### PR TITLE
Fix deprecated commands (set-env, add-path)

### PR DESCRIPTION
**Pull request by christopher-dG**
Finally got around to it 😅

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -54,11 +60,21 @@ const GHA = GitHubActions
...
-        @test ENV["a"] == "b"
-        @test (@capture_out set_env("a", ())) == "::set-env name=a::[]\n"
-        @test ENV["a"] == "[]"
+    mktemp() do file, io
+        withenv("GITHUB_ENV" => file, map(c -> string(c) => nothing, 'a':'z')...) do
+            set_env("a", "b")
+            @test ENV["a"] == "b"
+            @test read(file, String) == "a<<EOF\nb\nEOF\n"
+            set_env("b", "fooEOFbar")
+            @test read(file, String) == "a<<EOF\nb\nEOF\nb<<EOFEOF\nfooEOFbar\nEOFEOF\n"
```
Comment (SaschaMann) [inline test/runtests.jl:69]: I was about to ask what happens in this case, good to see a test for it. But do you know how GitHub would handle it?

**Review by SaschaMann — APPROVED** [review]
It's a strange format. I have one question but otherwise it looks good to me!

**Comment by SaschaMann** [comment]
The tests appear to be stuck

File: test/runtests.jl (hunk 1 above)
Comment (christopher-dG) [reply test/runtests.jl:69]: It should handle any case except for variable names that have `<<` in them, since the delimiter changes to `EOFEOF` if there's an `EOF` in the variable.

For that last `<<` edge case, I actually have no idea but I also know that `@actions/core` doesn't handle it either. I don't even know what "properly" means for that example 😅

Comment (SaschaMann) [reply test/runtests.jl:69]: > I don't even know what "properly" means for that example 😅

I'd guess that's debatable :D 

If this follows `@actions/core`, it's probably fine

**Comment by christopher-dG** [comment]
Normally the tests fail on nightly because Cassette doesn't work there, but I've never seen it hang before. I can reproduce the hang locally too. But since it's only a test problem and not a package problem, I'm gonna go ahead and merge.
